src/MongoDB.Driver.Core/Core/NativeLibraryLoader/PinnedBuffer.cs

```diff
@@ -0,0 +1,49 @@
+﻿/* Copyright 2019–present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace MongoDB.Driver.Core.NativeLibraryLoader
+{
+    internal class PinnedBuffer : IDisposable
+    {
+        private GCHandle _handle;
```

Comment:
Can `_handle` be `readonly` also?

Reply:
Interesting... I got no such warning but then I don't use Resharper.

You can leave off the `readonly` then.